src/compas_rhino/geometry/brep/brep.py

```diff
@@ -298,6 +299,25 @@ def contains(self, object):
             return self._brep.IsPointInside(point_to_rhino(object), TOLERANCE, False)
         else:
             raise NotImplementedError
+    
+    def to_meshes(self, u=16, v=16):
+        """Convert the faces of this Brep shape to meshes.
+
+        Parameters
+        ----------
+        u : int, optional
+            The number of mesh faces in the U direction of the underlying surface geometry of every face of the Brep.
+        v : int, optional
+            The number of mesh faces in the V direction of the underlying surface geometry of every face of the Brep.
+
+        Returns
+        -------
+        list[:class:`~compas.datastructures.Mesh`]
+
+        """
+        rg_meshes = Rhino.Geometry.Mesh.CreateFromBrep(self._brep.native_brep, Rhino.Geometry.MeshingParamaters.Default)
```

Comment:
Notice `self._brep` is already the `Rhino.Geometry.Brep` instance so here you should use either `self._brep` directly or `self.native_brep`.